src/coreclr/jit/targetwasm.h

```diff
@@ -72,26 +76,29 @@
 
   #define RBM_ALLINT               RBM_R0
 
+  #define RBM_CALL_GC_REGS_ORDER   RBM_NONE
+
   #define CNT_CALLEE_SAVED         1
   #define CNT_CALLEE_TRASH         0 // This and below are only used for CSE heuristics; thus an optimistic estimate for an "average" target.
   #define CNT_CALLEE_ENREG         8
 
   #define CNT_CALLEE_SAVED_FLOAT   8
   #define CNT_CALLEE_TRASH_FLOAT   4
+  #define CNT_CALL_GC_REGS         (CNT_CALLEE_SAVED) // This is how targetx86 is defined.
 
-  #define REG_CALLEE_SAVED_ORDER   REG_R0
+  #define REG_CALLEE_SAVED_ORDER   RBM_R0
   #define RBM_CALLEE_SAVED_ORDER   RBM_R0
 
   // GenericPInvokeCalliHelper VASigCookie Parameter
-  #define REG_PINVOKE_COOKIE_PARAM REG_R0
+  #define REG_PINVOKE_COOKIE_PARAM RBM_R0
```

Comment:
There are a number of cases where `REG_` values have been changed to be defined as `RBM_R0`, which are masks. They should be changed back to register values (`REG_NA` if that works, brought back `R0/F0` otherwise).

Reply:
Reinstated `R0`